backends/dpdk/backend.cpp

```diff
@@ -50,16 +55,16 @@ void PsaSwitchBackend::convert(const IR::ToplevelBlock *tlb) {
     auto rewriteToDpdkArch =
         new DPDK::RewriteToDpdkArch(refMap, typeMap, &collector);
     PassManager simplify = {
+        new DpdkArchFirst(),
         new P4::EliminateTypedef(refMap, typeMap),
         // because the user metadata type has changed
         new P4::ClearTypeMap(typeMap),
         new P4::TypeChecking(refMap, typeMap),
+        // TBD: implement dpdk lowering passes instead of reusing bmv2's
```

Comment:
We should strive to avoid duplicate code if possible, so if this needs some refactoring we should consider doing it.